backend/src/core/services/file-extensions.service.ts

```diff
@@ -0,0 +1,58 @@
+import FileType from 'file-type'
```

Comment:
Can we combine this module with `file-attachment.service` as I don't think this is a big enough usecase to deserve a dedicated module, it should just be part of `file-attachment.service`

Reply:
To push back against this, I think it's good to separate the concerns (checking file type) vs parsing the actual file. This also allows our test modules to be smaller and more targeted.